justfile

```
@@ -13,8 +13,10 @@ wait-for-ingress-controller:
 deploy-namespace:
   kubectl apply -f kubernetes/namespace.yml
 
-deploy-celestia-local:
-  kubectl apply -k kubernetes/celestia-local
+celestia-default: "local"
```

I think this syntax is slightly wrong. https://github.com/casey/just#recipe-parameters
```suggestion
celestia-default := 'local'
```